core/DataAccess/ArchiveSelector.php

```diff
@@ -67,15 +68,28 @@ public static function getArchiveIdAndVisits(ArchiveProcessor\Parameters $params
         $doneFlags      = Rules::getDoneFlags($plugins, $segment);
         $doneFlagValues = Rules::getSelectableDoneFlagValues();
 
-        $results = self::getModel()->getArchiveIdAndVisits($numericTable, $idSite, $period, $dateStartIso, $dateEndIso, $minDatetimeIsoArchiveProcessedUTC, $doneFlags, $doneFlagValues);
-
+        $results = self::getModel()->getArchiveIdAndVisits($numericTable, $idSite, $period, $dateStartIso, $dateEndIso, null, $doneFlags, $doneFlagValues);
         if (empty($results)) {
             return false;
         }
 
-        $idArchive = self::getMostRecentIdArchiveFromResults($segment, $requestedPlugin, $results);
+        list($idArchive, $tsArchived) = self::getMostRecentIdArchiveFromResults($segment, $requestedPlugin, $results);
```

Comment:
Wouldn't you need to make sure to use the time of the `start` flag here?

Reply:
I mean just to be safe. Cause even archiving one plugin can take 1 hour.

Reply:
Think it could be easy to do, will change this.